### DISCUSSION TITLE
I'm really enjoying the S-132.

### DISCUSSION THREAD

**Discussion by nbreeden2** [thehighnibble/thehighnibble.github.io#31] [S-132 add-on for IMSAI & Cromemco replicas]
I'm really enjoying the S-132. I have a monitor on a stand above the IMSAI and a keyboard sitting in front of it. It feels very much like I'm using a VT100 compatible terminal from back in the day. The S-132 adds additional screens that deal with the S-132 hardware such as the D+7AIO analog I/Os and physical joysticks. At the moment I'm programming in C directly on the IMSAI under CPM 2.2 and ran into an issue. It was amazing to drop to the front panel and enter a small program to help debug the issue. 
-Neil

**Comment by dmcnaugh** [comment]
Great, thanks for the feedback and for all your input during the testing.
